mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJsonTest.java

```diff
@@ -46,36 +46,35 @@
 import org.apache.james.mailbox.store.mail.model.impl.PropertyBuilder;
 import org.apache.james.mailbox.store.mail.model.impl.SimpleMailboxMessage;
 import org.apache.james.mailbox.tika.TikaConfiguration;
-import org.apache.james.mailbox.tika.TikaContainerSingletonRule;
+import org.apache.james.mailbox.tika.TikaExtension;
 import org.apache.james.mailbox.tika.TikaHttpClientImpl;
 import org.apache.james.mailbox.tika.TikaTextExtractor;
 import org.apache.james.metrics.api.NoopMetricFactory;
 import org.apache.james.util.ClassLoaderUtils;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 import com.google.common.collect.ImmutableList;
 
-public class MessageToElasticSearchJsonTest {
-    private static final int SIZE = 25;
-    private static final int BODY_START_OCTET = 100;
-    private static final TestId MAILBOX_ID = TestId.of(18L);
-    private static final MessageId MESSAGE_ID = TestMessageId.of(184L);
-    private static final ModSeq MOD_SEQ = ModSeq.of(42L);
-    private static final MessageUid UID = MessageUid.of(25);
-    private static final Username USERNAME = Username.of("username");
+class MessageToElasticSearchJsonTest {
+    static final int SIZE = 25;
+    static final int BODY_START_OCTET = 100;
+    static final TestId MAILBOX_ID = TestId.of(18L);
+    static final MessageId MESSAGE_ID = TestMessageId.of(184L);
+    static final ModSeq MOD_SEQ = ModSeq.of(42L);
+    static final MessageUid UID = MessageUid.of(25);
+    static final Username USERNAME = Username.of("username");
 
-    private TextExtractor textExtractor;
+    TextExtractor textExtractor;
+    Date date;
+    PropertyBuilder propertyBuilder;
 
-    private Date date;
-    private PropertyBuilder propertyBuilder;
+    @RegisterExtension
+    static TikaExtension tika = new TikaExtension();
```

Comment:
does `TikaExtension` hold a singleton docker tika?

Reply:
Nope... good point :)

Reply:
The fix is related to an other PR though, where I introduced `TikaExtension` (see #2964) in this commit : 
https://github.com/linagora/james-project/pull/2964/commits/d8b7fec43b2e9c6af06d9d5690bb52dd4803415a